### PR TITLE
make a drive show feature

### PR DESCRIPTION
**Pull request by Simranmalviya123**

### CODE REVIEW

File: src/drive/DisplayCard.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export default function DisplayCard({ driveCardDetails }: DisplayCardProps) {
+  return (
+    <>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: 'auto auto auto auto',
+          gap: '2rem'
```
Comment (shivamgupta2017) [inline src/drive/DisplayCard.tsx]: make it using style module

File: src/drive/Drive.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import video from '../image/video.png';
+import image from '../image/image.png';
+import natureimage from '../image/nature.webp';
+import zipimage from '../image/zip.png';
+import wallpaper from '../image/wallpaper.jpg';
+import filefolder from '../image/filefolder.png';
+
+const driveCardDetails = [
+  { title: 'audio.mp3', image: audio },
+  { title: 'video.mp4', image: video },
```
Comment (shivamgupta2017) [inline src/drive/Drive.tsx]: put this content in separate constant file

File: src/drive/DisplayCard.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { Box } from '@mui/material';
+import * as React from 'react';
+import DriveCard from './DriveCard';
+interface DisplayCardProps {
+  driveCardDetails: Array<{ title: string, image: string}>
```
Comment (shivamgupta2017) [inline src/drive/DisplayCard.tsx]: add this interface in separate file of interface.

File: src/drive/DisplayCard.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+export default function DisplayCard({ driveCardDetails }: DisplayCardProps) {
+  return (
+    <>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: 'auto auto auto auto',
+          gap: '2rem'
+        }}>
+        {driveCardDetails.map((cardDetails) => {
```
Comment (shivamgupta2017) [inline src/drive/DisplayCard.tsx]: add interface here.

File: src/drive/DriveCard.tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        sx={{ position: 'relative' }}
+        aria-owns={anchorEl ? 'simple-menu' : undefined}
+        aria-haspopup="true"
+        onMouseEnter={mouseInHandler}
+        onMouseLeave={mouseOutHandler}>
+        <CardMedia sx={{ height: 110 }} image={cardDetails.image} title={cardDetails.title} />
+
+        <CardContent
+          sx={{
+            height: '10px',
```
Comment (shivamgupta2017) [inline src/drive/DriveCard.tsx]: pull these css using css-module.

File: src/drive/Constant.ts [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```
Comment (rraj77) [inline src/drive/Constant.ts]: remove unnecessary imports

**Pull request merged**